From bf767f15b586282410f356ba4fe9182c925c586a Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Tue, 28 Oct 2025 08:23:27 +0300
Subject: [PATCH 521/553] QTest: include <chrono> instead of relying on
 transitive includes

Found in code review.

Pick-to: 6.8 6.5
Change-Id: I111907d6b365ea642c44dff6caed65192124dff9
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
(cherry picked from commit 6cbe30f900fce4c757701ef3b72fb5868bcb7dd4)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/testlib/qtestcase.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/testlib/qtestcase.h b/src/testlib/qtestcase.h
index 4b01fa656c3..82d7b353bde 100644
--- a/src/testlib/qtestcase.h
+++ b/src/testlib/qtestcase.h
@@ -15,6 +15,7 @@
 #include <QtCore/qtemporarydir.h>
 #include <QtCore/qthread.h>
 
+#include <chrono>
 #ifdef __cpp_concepts
 #include <concepts>
 #endif
-- 
2.51.2

